src/borg/helpers/parseformat.py

```diff
@@ -662,14 +665,26 @@ def archive(self):
 
     def get_meta(self, key, rs):
         value = self.archive.metadata.get(key, '')
-        return remove_surrogates(value) if rs else value
+        if rs:
+            if isinstance(value, list):
+                return self.format_cmdline(map(remove_surrogates, value))
+            else:
+                return remove_surrogates(value)
+        else:
+            return value
```

Comment:
also, the else branch just passes through the list.

works now, but not in general if we want to support lists.